port/zephyr/golioth_version_boot_msg.c

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2024 Golioth, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/init.h>
+#include <golioth_version.h>
+
+#define GOLTIOTH_BOOT_BANNER_PRIO 32
+int golioth_boot_banner(void)
+{
+    printk("*** Golioth Firmware SDK %s ***\n",
+           (IS_ENABLED(CONFIG_GOLIOTH_VERSION_INCLUDE_COMMIT)) ? STRINGIFY(GOLIOTH_BUILD_VERSION)
```

Comment:
Is `GOLIOTH_BUILD_VERSION` defined? I don't see that configured in cmake.

Reply:
It is automatically generated by Zephyr's gen_version_h.cmake: 

https://github.com/zephyrproject-rtos/zephyr/blob/948fbacd33be41f6623e8ef6f1211e93ba4a7237/cmake/gen_version_h.cmake#L13